### DISCUSSION TITLE
Running on forks

### DISCUSSION THREAD

**Discussion by DanySK** [renovatebot/renovate#13430] [Request Help] [answered]
Hi, I need to run on a fork. From the docs, my understanding was that the following:

```json
{
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
      "github>DanySK/renovate-config"
   ],
   "includeForks": true
}
```

Would have triggered renovate runs on my forked repo. However, looking into the dashboard, all run results are marked as `fork`, and from the log it behaves like if I have not configured it manually:

```
DEBUG: No dangling containers to remove
INFO: Repository started
{
  "renovateVersion": "31.21.0"
}
DEBUG: Using localDir: /mnt/renovate/gh/DanySK/gson-extras
DEBUG: Repository cache not found
{
  "cacheFileName": "/tmp/renovate-cache/renovate/repository/github/DanySK/gson-extras.json"
}
DEBUG: initRepo("DanySK/gson-extras")
DEBUG: Overriding default GitHub endpoint
{
  "endpoint": "https://api.github.com/"
}
DEBUG: DanySK/gson-extras default branch = master
DEBUG: Using app token for git init
INFO: Repository is a fork and not manually configured - skipping
DEBUG: Repository timing splits (milliseconds)
{
  "splits": {},
  "total": 957
}
DEBUG: http statistics
{
  "urls": {
    "https://api.github.com/graphql (POST)": 1,
    "https://api.github.com/repos/DanySK/gson-extras/contents/renovate.json (GET)": 1
  },
  "hostStats": {
    "api.github.com": {
      "requestCount": 2,
      "requestAvgMs": 189,
      "queueAvgMs": 0
    }
  },
  "totalRequests": 2
}
INFO: Repository finished
{
  "durationMs": 957
}
```

**Comment by rarkins** [comment, marked as answer]
File name might need to be renovate.json when it's a fork

**Reply by DanySK** [reply]
The name is `renovate.json`, the exact repo and file are: https://github.com/DanySK/gson-extras/blob/master/renovate.json

**Reply by DanySK** [reply]
@rarkins is there any way I can trigger a re-run of renovate to check if the problem is still reproducible? In case, I think I should fill a bug report.
Also: is there a way to inspect the final configuration on the dashboard? To be sure that the final configuration resulting from multiple extends is compiled correctly.

Thanks.

**Reply by rarkins** [reply]
If you have installed "Selected repos" then the easiest way is to uninstall it, save, wait a few seconds, then install it again and save. If you have instead "All repos" then I'll need to trigger it manually

**Reply by DanySK** [reply]
I'm using "all repos" (I'm using renovate on dozens of small FOSS utilities).

Suggestion: it might make sense to have the option to trigger a run from the dashboard just like it is possible from the dashboard issue checkbox.

**Reply by rarkins** [reply]
Yes, we hope to add that feature one day, but the dashboard needs a rewrite first

**Reply by rarkins** [reply]
Your renovate.json doesn't parse:

![image](https://user-images.githubusercontent.com/6311784/148751072-3939c2f1-931d-42b8-bda7-411d89dda411.png)

**Reply by DanySK** [reply]
Ugh! Thanks! I'm sorry for the silly mistake.

I'm so used to trailing commas that I sometimes fail at removing them in JSON. Also, I picked up the habit to have the onboarding PR tell me when my configuration is mistaken, but that clearly could not apply to this very case.

Let's see if that was the culprit...

**Reply by DanySK** [reply]
That was it, indeed. Thanks, sorry again for my stupid mistake.
